tests/test_observation_resource.py

```diff
@@ -0,0 +1,213 @@
+#                 "coding": [
+#                     {
+#                         "system": "http://loinc.org",
+#                         "code": "8462-4",
+#                         "display": "Diastolic blood pressure",
+#                     }
+#                 ]
+#             },
+#             "valueQuantity": {
+#                 "value": 60,
+#                 "unit": "mmHg",
+#                 "system": "http://unitsofmeasure.org",
+#                 "code": "mm[Hg]",
+#             },
+#             "interpretation": [
+#                 {
+#                     "coding": [
+#                         {
+#                             "system": "http://terminology.hl7.org/CodeSystem/v3-ObservationInterpretation",  # noqa: E501
+#                             "code": "L",
+#                             "display": "low",
+#                         }
+#                     ],
+#                     "text": "Below low normal",
+#                 }
+#             ],
+#         },
+#     ],
+# }
+
+OBSERVATION_DICT_INPUT = {
+    "resourceType": "Observation",
+    "status": "final",
+    "category": [
+        {
+            "coding": [
+                {
+                    "system": "http://terminology.hl7.org/CodeSystem/observation-category",  # noqa: E501
+                    "code": "vital-signs",
+                    "display": "Vital Signs",
+                }
+            ]
+        }
+    ],
+    "code": {
+        "coding": [
+            {
+                "system": "http://loinc.org",
+                "code": "85354-9",
+                "display": "Blood pressure panel with all children optional",
+            }
+        ],
+        "text": "Blood pressure systolic & diastolic",
+    },
+    "subject": {"reference": "Patient/example"},
+    "effectiveDateTime": "2012-09-17",
+    "performer": [{"reference": "Practitioner/example"}],
+    "interpretation": [
+        {
+            "coding": [
+                {
+                    "system": "http://terminology.hl7.org/CodeSystem/v3-ObservationInterpretation",  # noqa: E501
+                    "code": "L",
+                    "display": "low",
+                }
+            ],
+            "text": "Below low normal",
+        }
+    ],
+    "bodySite": {
+        "coding": [
+            {
+                "system": "http://snomed.info/sct",
+                "code": "368209003",
+                "display": "Right arm",
+            }
+        ]
+    },
+}
+
+OBSERVATION_FLAT = {
+    "resourceType": "Observation",
+    "category.code": "http://terminology.hl7.org/CodeSystem/observation-category|vital-signs",  # noqa: E501
+    "category.text": "Vital Signs",
+    "effectiveDateTime": datetime.date(2012, 9, 17),
+    "performer": "Practitioner/example",
+    "interpretation.code": "http://terminology.hl7.org/CodeSystem/v3-ObservationInterpretation|L",  # noqa: E501
+    "interpretation.text": "Below low normal",
+    "code.code": "http://loinc.org|85354-9",
```

Comment:
Are we assuming only one coding for observation, unlike Condition?

Reply:
No, there can be one or multiple. This example only had one (I just pulled something from the FHIR examples)